<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_NUM3">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_equal">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_NUM9">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_plus">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_NUM1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_NUM5">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_NUM2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_NUM8">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_rem_from_division">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_NUM7">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_NUM0">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>145</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>145</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_minus">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_AC">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_dot">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_NUM4">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_NUM_sign">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_NUM6">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_multiply">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>х</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_divide">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>70</width>
         <height>55</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #808080; 

font: 600 17pt &quot;Segoe UI&quot;; 

border-radius: 8px; 

gridline-color: rgb(0, 0, 0); 

color: #ffffff; </string>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 

font: 600 20pt &quot;Segoe UI&quot;;

border-radius: 8px; 


color: #ffffff; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_Error">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
